src/lib/tvgLoadModule.h

```diff
@@ -30,11 +30,7 @@ namespace tvg
 class LoadModule
 {
 public:
-    //default view box, if any.
-    float vx = 0;
```

Comment:
Please fix the only wrong convention. otherwise, there is no point this kind of changes.

I think you can just move "w/h" to svg side instead of width/height.

Reply:
in the loader we have:
- 'vx, vy, vw, vh' as the viewbox parameters
- and 'w, h' that are the width and height for all types of images, except the svgs (so I don;t want to move these two from the loadModule to the svg loader)
For the svg file, the real width and height of the image are the width and height from the 'viewbox' attribute. at least it is how I understand it... 


For now there are still some issues regarding this subject, but it's a different story I think:
```
<!DOCTYPE html>
<html>

<head>
	<title>SVG viewBox Attribute</title>
	<style type="text/css">
		svg {
			border: 1px solid #aaa;
		}
	</style>
</head>

<body>

	<svg width="200" height="200">
		<circle cx="50" cy="50" r="45" stroke="#000" stroke-width="3" fill="none"/>
	</svg>
    
	<svg>
		<circle cx="50" cy="50" r="45" stroke="#000" stroke-width="3" fill="none"/>
	</svg>
	
	<svg width="200" height="200" viewBox="0 0 100 100">
		<circle cx="50" cy="50" r="45" stroke="#000" stroke-width="3" fill="none"/>
	</svg>
 
        <svg viewBox="0 0 100 100">
		<circle cx="50" cy="50" r="45" stroke="#000" stroke-width="3" fill="none"/>
	</svg>
    
       <svg viewBox="0 0 200 200">
		<circle cx="50" cy="50" r="45" stroke="#000" stroke-width="3" fill="none"/>
	</svg>
</body>

</html>
```